TWI-15 Follow/unfollow

pm888 opened the pull request:
1. added methods Follow/unfollow
2. added methods Follow/unfollow into openapi.yaml
3. added HandleFunc for ResetPassword
4. deleted UsersFollow struct

pm888 committed:
TWI-15 Follow/unfollow
1. added methods Follow/unfollow
2. added methods Follow/unfollow into openapi.yaml
3. added HandleFunc for ResetPassword
4. deleted UsersFollow struct

bunin commented on internal/server/server.go:
```diff
@@ -12,27 +12,19 @@ func Server() {
+	r.HandleFunc("/v1/users/reset-password", func(w http.ResponseWriter, r *http.Request) {
+		Serviceuser.AuthHandler(http.HandlerFunc(Serviceuser.ResetPassword)).ServeHTTP(w, r)
+	}).Methods(http.MethodPost)
+	r.HandleFunc("/v1/users/follow", func(w http.ResponseWriter, r *http.Request) {
```
this probably should be `/v1/users/{id}/follow`?

bunin commented on internal/server/server.go:
```diff
@@ -12,27 +12,19 @@ func Server() {
+	r.HandleFunc("/v1/users/follow", func(w http.ResponseWriter, r *http.Request) {
+		Serviceuser.AuthHandler(http.HandlerFunc(Serviceuser.FollowUser)).ServeHTTP(w, r)
+	}).Methods(http.MethodPost)
+	r.HandleFunc("/v1/users/unfollow", func(w http.ResponseWriter, r *http.Request) {
```
ditto

pm888 replied:
corrected. thanks.

pm888 committed:
TWI-15 Follow/unfollow
1. added methods Follow/unfollow
2. added methods Follow/unfollow into openapi.yaml
3. added HandleFunc for ResetPassword
4. deleted UsersFollow struct
